docs/Quickstart/connecting-to-the-outside.md

```diff
@@ -13,7 +13,7 @@ Telegram is a great starting point for a chat platform because it has a free API
 We will use an existing integration with Telegram as a way for your system to communicate with the outside world. Just install the telegram adapter:
 
 ```
-root:/app$ pip install "telegram-adapter[dev]==4.0.0a7"
+root:/app$ pip install telegram-adapter==4.0.0a9
```

Comment:
We still need the `[dev]` part everywhere. Without it you can't run the `setup` command

Reply:
The quotes are recommended too, because some shells interpret brackets differently